vyxal/elements.py

```diff
@@ -3023,6 +3023,60 @@ def multiply(lhs, rhs, ctx):
         }.get(ts, lambda: vectorise(multiply, lhs, rhs, ctx=ctx))()
 
 
+def multiset_difference(lhs, rhs, ctx):
+    """Element Þ∨
+    (lst, lst) -> Return the mutli-set difference of two lists
+    """
+
+    lhs = iterable(lhs, ctx=ctx)
+    lhs_copy = deep_copy(lhs)
+    rhs = iterable(rhs, ctx=ctx)
+
+    for item in rhs:
+        if item in lhs_copy:
+            lhs_copy.remove(item)
+
+    return lhs_copy
+
+
+def multiset_intersection(lhs, rhs, ctx):
+    """Element Þ∩
+    (lst, lst) -> Return the multi-set intersection of two lists
+    """
+
+    lhs = iterable(lhs, ctx=ctx)
+    rhs = iterable(rhs, ctx=ctx)
+    res = []
+
+    for item in rhs:
+        if item in lhs:
+            res.append(item)
+
+    return res
```

Comment:
This doesn't seem to handle the multiset part - it looks like passing `[1, 1, 1]` and `[1]` will return `[1, 1, 1]` where it should return `[1]`.